rustenv.py

```diff
@@ -31,11 +31,11 @@
 
 # TODO: it might be nice to intelligently add/remove from PATH/PS1 instead of
 # just restoring the old one
-_RUSTENV_OLD_PS1="$PS1"
+_RUSTENV_OLD_PS1="${{PS1:-}}"
```

Comment:
virtualenv uses just `${PS1-}` and has some additional logic for unset PS1 -- maybe take that for inspiration?

Reply:
I looked at virtualenv and they use `${PS1:-}` (python3.10). I need the double curly brackets because we pass the string into format. I am not sure what you would like me to change for that.

As for the additional logic, I can implement that, should I go ahead and implement the same logic for all the variables in the deactivate function?

Reply:
I believe you're looking at `venv` -- not `virtualenv`